satpy/readers/__init__.py

```diff
@@ -475,7 +496,7 @@ def find_files_and_readers(start_time=None, end_time=None, base_dir=None,
     for reader_configs in configs_for_reader(reader, ppp_config_dir):
         try:
             reader_instance = load_reader(reader_configs, **reader_kwargs)
-        except (KeyError, MalformedConfigError, yaml.YAMLError) as err:
+        except (KeyError, IOError, yaml.YAMLError) as err:
```

Comment:
Where does the `IOError` come from?

Reply:
If the file being read doesn't open for some reason (FileNotFoundError, etc).